src/Homework/Week3/Pb3/testPb3.java

```diff
@@ -0,0 +1,74 @@
+package Homework.Week3.Pb3;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class testPb3 {
+
+    public static void main(String[] args) {
+        ArrayList<User> allUsers = new ArrayList<User>();
+        Board board = new Board();
+        Scanner s = new Scanner(System.in);
+        User currentUser = null;
+        while(true)
+        {
+            System.out.print("Enter a command and press enter: ");
+            String command = s.nextLine();
+            if(command.equals("register"))
+            {
+                System.out.print("Name: ");
+                String name = s.nextLine();
+                System.out.print("Email: ");
+                String email = s.nextLine();
+                System.out.print("Password: ");
+                String password = s.nextLine();
+                currentUser = new User(name, email, password);
+                if(!allUsers.contains(currentUser))
```

Comment:
cred ca lipseste o implementare pe equals in User; daca vei crea un user cu acelasi name/email/password ca cele ale unui user deja existent, il va vedea cu un user nou si va fi adaugat in lista, chiar daca tu ai adaugat conditia asta